src/components/display.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import '../App.css';
+import { useDispatch } from 'react-redux';
+import PropTypes from 'prop-types'; // eslint-disable-line import/no-extraneous-dependencies
```

Comment:
- [ ] Kindly remove this eslint disable line it's best to fix the issue in the code.